capstone/src/main/webapp/js/calendar.js

```diff
@@ -0,0 +1,66 @@
+/* eslint no-unused-vars: ["error", {"varsIgnorePattern": "drawCharts"}] */
+
+/**
+ * Callback function to generate charts.
+ */
+function drawCharts() {
+  const frequencyData = new google.visualization.DataTable();
+  fetch('/viz/deadlines').
```

Comment:
No action needed for this PR but please file an issue to later deal with server errors during fetch. It'll be a short and sweet stretch goal. This page seems useful: https://learnwithparam.com/blog/how-to-handle-fetch-errors/.